library/src/main/java/com/google/maps/android/StreetViewUtil.kt

```diff
@@ -35,14 +35,24 @@ class StreetViewUtils {
         /**
          * This function will check whether a location is available on StreetView or not.
          *
-         * @param latLng Location to check
-         * @param apiKey Maps API Key
-         * @return A Status value specifying if the location is available on Street View or not,
-         * whether the used key is a right one, or any other error.
+         * @param latLng The `LatLng` object representing the location for which you want to fetch Street View data.
+         * @param apiKey The API key for Google Maps services.
+         * @param source The source of the Street View panorama. It can be one of the following values:
```

Comment:
Indicate that the source parameter is optional because it has a default value

Reply:
Done ✅